### ISSUE TITLE
[Screen reader - MyAccess - Filter] Buttons must have discernible text (.ms-Button--icon) on filters screen.​  

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#10338]
**User Experience:**
User would find difficulty to identify the controls if name is not defined for that control as user won't understand the purpose of such control and also he/she would surely skip the functionality over that control.

**Repro steps:**
1. Open URL 'https://myaccess.microsoft.com/@M365x202152.onmicrosoft.com#/access-packages' and enter valid credentials.
2. Navigate to 'Request History' list item and select it.​
3. 'Request History' screen will open.​
4. Navigate to ' Filter' button and select it.​
5. 'Filters' screen will open.
6. Navigate to cancel control on filters screen.
7. Verify name of cancel control.

**Actual Result:**
Name is not defined for 'cancel' control on filters screen under Request history section and due to this user is unable to identify the context of that control.

**Expected Result:**
Name should be defined as 'Cancel' for 'cancel' control on filters screen under request history section so that end user could get the context and preform functionality over such controls.

**Note:** 
1. This issue also exists under Request History->View Details->Access Details for Cancel control.
2. This issue also exists with chrome + JAWS.
3. This issue also exists under Access Package->Expired->Renew Access for cancel control.

Issue: Buttons must have discernible text (button-name: https://dequeuniversity.com/rules/axe/3.1/button-name?application=axeAPI)
​
Target application title: My Access​
Target application url: https://myaccess.microsoft.com/@M365x202152.onmicrosoft.com#/request-history​
​
**Element path:** .ms-Button--icon​
​
**Snippet**: <button type="button" class="ms-Button ms-Button--icon root-252" data-is-focusable="true"><div class="ms-Button-flexContainer flexContainer-58"><i data-icon-name="Cancel" class="ms-Button-icon icon-253" role="presentation"></i></div></button>​
​
**How to fix:​**
Fix all of the following:​
  Element is in tab order and does not have accessible text​
​
**Fix any of the following:​**
  Element has a value attribute and the value attribute is empty​
  Element has no value attribute or the value attribute is empty​
  Element does not have inner text that is visible to screen readers​
  aria-label attribute does not exist or is empty​
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty​
  Element's default semantics were not overridden with role="presentation"​
  Element's default semantics were not overridden with role="none"​
  Element has no title attribute or the title attribute is empty

This accessibility issue was found using Accessibility Insights for Web 2.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights 

**MAS Reference:** 
MAS 4.1.2 - Name, Role, Value

![MAS4 1 2_Request History_Filter_Name is not defined](https://user-images.githubusercontent.com/30456910/64168731-c005bc00-ce69-11e9-8043-c5e303296df6.png)
![MAS4 1 2_Request History_Access details_Name is not defined](https://user-images.githubusercontent.com/30456910/64168730-c005bc00-ce69-11e9-919b-897a4fb71aa5.png)
![MAS4 1 2_Access Package_Expired_Renew Access_Name is not defined](https://user-images.githubusercontent.com/30456910/64168733-c09e5280-ce69-11e9-892e-a125020134d1.png)

**Comment by KevinTCoughlin** [comment]
It looks like the validation error flagged here is that the "Close" button in a Panel component does not have discernible text since it is an icon character. I will need to dig in and see what the appropriate fix is if any.

**Comment by dzearing** [comment]
@KevinTCoughlin Any updates?

**Comment by dzearing** [comment]
This is an app user error. The `Panel` component has a prop `closeButtonAriaLabel` which is not being provided; hence no aria-label is placed on the close button.

**Comment by dzearing** [comment]
If you feel this solves your question, please close the issue. Thanks.
